riot-basics/gpio/README.md

```diff
@@ -16,6 +16,19 @@ used to toggle an on-board LED.
 
 **Objective:**
 
+- Add the GPIO's feature to the requirements of the application (in the `Makefile`):
```

Comment:
You could also remove the line "Objective" above. I think this is a leftover from previous versions.